spec/models/route_spec.rb

```diff
@@ -32,4 +26,40 @@
       expect(create(:route)).to be_valid
     end
   end
+
+  describe 'methods' do
```

Comment:
⛏️ `describe '#stop'` already signals that this is an instance method because of the `#` sign. If it's a class method, your describe block would be `describe '.stop'`. In either case, you don't need to nest this under another describe.